arcgis-ios-sdk-samples/Route & Navigation/Offline routing/OfflineRoutingViewController.swift

```diff
@@ -73,38 +73,38 @@ class OfflineRoutingViewController: UIViewController, AGSGeoViewTouchDelegate {
         
         //add graphics overlay, one for the stop graphics
         //and other for the route graphics
-        self.mapView.graphicsOverlays.addObjectsFromArray([self.routeGraphicsOverlay, self.stopGraphicsOverlay])
+        self.mapView.graphicsOverlays.addObjects(from: [self.routeGraphicsOverlay, self.stopGraphicsOverlay])
         
         //get the path for the geodatabase in the bundle
-        let dbPath = NSBundle.mainBundle().pathForResource("sandiego", ofType: "geodatabase", inDirectory: "san-diego")!
+        let dbPath = Bundle.main.path(forResource: "sandiego", ofType: "geodatabase", inDirectory: "san-diego")!
         
         //initialize the route task using the path and the network name
-        self.routeTask = AGSRouteTask(fileURLToDatabase: NSURL(fileURLWithPath: dbPath), networkName: "Streets_ND")
+        self.routeTask = AGSRouteTask(fileURLToDatabase: URL(fileURLWithPath: dbPath), networkName: "Streets_ND")
         
         //get default route parameters
         self.getDefaultParameters()
         
         //zoom to San Diego
-        self.mapView.setViewpointCenter(AGSPoint(x: -13042254.715252, y: 3857970.236806, spatialReference: AGSSpatialReference(WKID: 3857)), scale: 2e4, completion: nil)
+        self.mapView.setViewpointCenter(AGSPoint(x: -13042254.715252, y: 3857970.236806, spatialReference: AGSSpatialReference(wkid: 3857)), scale: 2e4, completion: nil)
         
         //enable magnifier for better experience while using tap n hold to add a stop
-        self.mapView.interactionOptions.magnifierEnabled = true
+        self.mapView.interactionOptions.isMagnifierEnabled = true
     }
     
     //method returns a graphic for the specified location
     //also assigns the stop number
-    private func graphicForLocation(point:AGSPoint) -> AGSGraphic {
+    fileprivate func graphicForLocation(_ point:AGSPoint) -> AGSGraphic {
         let symbol = self.symbolForStopGraphic(self.stopGraphicsOverlay.graphics.count + 1)
         let graphic = AGSGraphic(geometry: point, symbol: symbol, attributes: nil)
         return graphic
     }
     
-    private func symbolForStopGraphic(index: Int) -> AGSSymbol {
+    fileprivate func symbolForStopGraphic(_ index: Int) -> AGSSymbol {
```

Comment:
similar to other comments above about having argument name to clarify call site usage